pipeline/e2e-test.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+steps:
+    - task: Npm@1
+      inputs:
+          command: custom
+          customCommand: run build:all
+      displayName: npm run build:all
+
+    # e2e tests depend on npm run drop
```

Comment:
Do the e2e tests really depend on `npm run drop`? Because we don't seem to be doing that.

Reply:
Comment is misleading. The actual dependency is on the `drop` folder, not on `npm run drop`. So, we need to run some of our steps to generate that `drop` folder and I guess the simplest, most straightforward way to do this is by running `npm run build:all`.

I updated the comment to reflect this.